.travis.mk

```diff
@@ -46,7 +46,7 @@ test_ubuntu: deps_ubuntu
 
 deps_osx:
 	brew update
-	brew install openssl readline curl icu4c --force
+	brew install openssl readline curl icu4c libiconv --force
```

Comment:
Hm. We don't experience any build problems on travis. Why did you added it here?